"Start" script is missing

# Description
The "npm start" command fails due to the script being missing from the config file.  Running "node src/bot.js" successfully starts the bot, however.

# Steps to Reproduce
Steps to reproduce the behaviour:
1. Navigate to the install directory in Command Prompt or other shell
2. Send command `npm start`

# Expected Behaviour
The bot should have started.

# Context
**Evidence:**
If applicable, include any screenshots or error messages to help explain your problem.

**Environment (please complete the following information):**
`npm ERR! missing script: start`

--- a/package.json
+++ b/package.json
@@ -6,6 +6,9 @@
   "keywords": [],
   "author": "Kay <kylrs00@gmail.com>",
   "license": "ISC",
+  "scripts": {
+    "start": "node src/bot.js"
+  },
   "dependencies": {
     "bufferutil": "^4.0.1",
     "discord.js": "^12.2.0",

--- a/src/modules/logging/events.js
+++ b/src/modules/logging/events.js
@@ -4,7 +4,7 @@
  * @author Kay <kylrs00@gmail.com>
  * @license ISC - For more information, see the LICENSE.md file packaged with this file.
  * @since r20.2.0
- * @version v1.0.2
+ * @version v1.0.3
  */
 
 const Logger = require('./Logger.js');
@@ -169,13 +169,46 @@ module.exports = {
      */
     async on_guildMemberRemove(client, member) {
         const user = member.user;
-        await client.logger.log(
-            member.guild,
-            'member',
-            `\`${user.username}#${user.discriminator}\` left the server.`,
-            '',
-            0x990000,
-            { 'Timestamp': `\`${new Date().toLocaleString()}\`` },
-        );
+
+        member.guild
+            .fetchAuditLogs({type: "MEMBER_KICK"})
+            .then(async (logs) => {
+                console.log(logs);
+                const now = new Date();
+                // Find a recent kick audit, targeting the user who just left
+                const kick = logs.entries.find((entry) => {
+                    return entry.target === user
+                        && now - entry.createdAt < 1e4;
+                });
+
+                if (kick) {
+                    await client.logger.log(
+                        member.guild,
+                        'member',
+                        `\`${user.username}#${user.discriminator}\` was kicked from the server.`,
+                        '',
+                        0x990044,
+                        {
+                            'By': kick.executor.toString(),
+                            'Timestamp': `\`${new Date().toLocaleString()}\``
+                        },
+                    );
+                } else {
+                    // If no kick audit was found, assume the user left of their own accord
+                    await client.logger.log(
+                        member.guild,
+                        'member',
+                        `\`${user.username}#${user.discriminator}\` left the server.`,
+                        '',
+                        0x990000,
+                        { 'Timestamp': `\`${new Date().toLocaleString()}\`` },
+                    );
+                }
+            })
+            .catch((err) => {
+                console.error(err);
+            });
+
+
     },
 };

